<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressWindow</class>
 <widget class="QWidget" name="ProgressWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tiến độ học tập</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Biểu đồ tiến độ học tập</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <!-- Vị trí sẽ chèn QChartView bằng code -->
   <item>
    <widget class="QWidget" name="chart_placeholder">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>200</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFF9C4; border: 2px dashed #F88379; border-radius: 15px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_comeback">
     <property name="text">
      <string>Quay lại</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFD1DC; border-radius: 10px; padding: 8px 20px; font-size: 16px;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>